helm/auto-deployment/configuration.md

```diff
@@ -0,0 +1,444 @@
+# Advanced Configuration
+
+## Client Libraries
+
+The following are the client libraries used in Bold BI by default.
+
+```console
+optionalLibs: 'mongodb,mysql,influxdb,snowflake,oracle,google,clickhouse'
+```
+
+Read the optional client library license agreement from the following link.
+
+[Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+
+If you wish to inlcude only specific client libraries note the optional client libraries from the above link as comma separated names in your values.yaml file.

```

Comment:
If you want to include only specific client libraries, note the optional client libraries from the above link as comma-separated names in your values.yaml file.

Reply:
concern addressed